src/css/components/_keyboard.scss

```
@@ -203,5 +203,6 @@
 }
 
 .text-security {
+    font-family: "LivioMask";
     -webkit-text-security: disc;
 }
```

The size of the masked characters is somewhat larger in firefox. It might be better to make the following change to make the masked characters consistent on both chromium and firefox
```suggestion
.text-security {
    font-family: "LivioMask";
    font-size: 18px;
}
```